### PR TITLE
Use `process.browser` directly

### PR DESCRIPTION
**Pull request by fanatid**
Issue #27

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ function isFunction(f) {
 
 module.exports = {
 
-  browser: process.browser,
+  browser: typeof process === "undefined" || !process.nextTick || Boolean(process.browser),
```
Comment (fanatid) [inline index.js]: I still don't understand why `process` can be undefined...
`!!(expression)` is much faster than `Boolean(expression)`

Comment (tinybike) [inline index.js]: It's less clear though, and this check only happens once so I don't see a need to optimize it for speed.  It's the same reason I thought that your version of `isCipherAvailable` that uses `.some` is preferable to the faster (but clunkier) version that just used a loop!

Comment (tinybike) [inline index.js]: I think `process` is supposed to be undefined in the browser -- open up a Chrome console and type `process`.  Is it defined?

Comment (fanatid) [inline index.js]: `browserify` defined `process`

**Pull request merged**